bolt-modules/boltlib/lib/puppet/functions/run_command.rb

```diff
@@ -13,6 +13,7 @@
   # @param options A hash of additional options.
   # @option options [Boolean] _catch_errors Whether to catch raised errors.
   # @option options [String] _run_as User to run as using privilege escalation.
+  # @option options [Hash] _env_vars Map of environment variables to set
```

Comment:
It looks like Puppet doesn't provide type validation for options and we don't provide it ourselves as part of the function. Is that something we'd want to do at some point?

Reply:
We could probably do it by setting the type to a `Struct` with appropriately-typed keys. I wasn't sure what sort of validation to do beyond Hash though. Requiring the values be strings seems overly restrictive, since numbers and booleans seem common and should work fine.

Reply:
Ya, I'm not sure that it's all that important, especially since it doesn't break anything. 🤷